packages/terra-form-fieldset/tests/jest/FormFieldset.test.jsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/jsx-boolean-value */
+
+import React from 'react';
+import Fieldset from '../../src/Fieldset';
+
+it('should render a default component', () => {
+  const field = (<Fieldset />);
+  const wrapper = shallow(field);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it('should render a Fieldset when all the possible props are passed into it', () => {
+  const input = (
+    <Fieldset
+      legend="Text"
```

Comment:
Let's add `className` prop to these as well. This is to help test that we are merging custom class names with the fieldset class names.